### ISSUE TITLE
[Suggestion] New section linking Everest projects and 3Box profiles

### ISSUE THREAD

**Issue by khangle27** [graphprotocol/mission-control-curator#197]
It'd be nice to have a new section below the project's summary showcasing 3Box profiles which are connected to the project with their role/title, twitter handle and other information (if made public). 

We already have the `User` field in the [Everest subgraph query](https://thegraph.com/explorer/subgraph/graphprotocol/everest). Linking it to a 3Box profile which may contain more information about the user and displaying that information on Everest.link would be nice.

**Comment by yanivtal** [comment]
Yes for sure. 3Box is coming out with their new [Ceramic](https://www.ceramic.network/) based identity system. I think we should wait for that and then absolutely.

**People** can have multiple **Profiles** and have **Roles** at **Projects**. Great example of relationships. We need subgraph composability before we can cleanly implement this relationship across subgraphs.
